features/step_definitions/ocm/ocm.rb

```diff
@@ -1,18 +1,30 @@
-Given /^I open ocm portal as an? #{WORD} user$/ do |usertype|
+Given /^I open ocm portal with #{WORD}$/ do |usertype|
   base_rules = BushSlicer::WebConsoleExecutor::RULES_DIR
   snippets_dir = BushSlicer::WebConsoleExecutor::SNIPPETS_DIR
+  portals = YAML.load_file(expand_private_path("config/credentials/ocm.yaml"))
   base_url = env.web_console_url
   step "I have a browser with:", table(%{
     | rules        | lib/rules/web/ocm_console/ |
     | base_url     | #{base_url}                |
     | snippets_dir | #{snippets_dir}            |
   })
-  if ocm_env == "stage"
-    browser.run_action(:goto_ocm_console)
-  elsif ocm_env == "product"
-    browser.run_action(:goto_ocm_console)
+  if base_url == "https://cloud.redhat.com/beta/openshift/"
```

Comment:
We should maintenance the web console URL at one place. And for you have set `ocm_stage` and `ocm_prod`,  you need use this to assign `ocm_env`